ote_sdk/ote_sdk/tests/usecases/evaluation/test_f_measure.py

```diff
@@ -1769,6 +1779,71 @@ def check_performance(performance, expected_score, expected_metrics):
                         == VisualizationType.TEXT
                     )
 
+        def generate_expected_default_dashboard_metric_groups(
+            actual_f_measure: FMeasure,
+        ):
+            return [
+                BarMetricsGroup(
+                    metrics=list(actual_f_measure.f_measure_per_label.values()),
+                    visualization_info=BarChartInfo(
+                        name="F-measure per label",
+                        palette=ColorPalette.LABEL,
+                        visualization_type=VisualizationType.RADIAL_BAR,
+                    ),
+                )
+            ]
+
+        def generate_expected_optional_dashboard_metric_groups(
+            actual_f_measure: FMeasure,
+        ):
+            per_confidence = CurveMetric(
```

Comment:
can not it be just a copy of `actual_f_measure.f_measure_per_confidence` ?

Reply:
It was in commit [https://github.com/openvinotoolkit/training_extensions/pull/845/commits/f1acdf954027fd9c481a3e0adb364e1fed730857](url)
but mypy linter is failed due to [list-item] errors [http://213.221.44.203/job/ote/job/precommit/job/ote_sdk_tests_runner/361/](url) . Should I rollback changes to this commit state and add “# type: ignore” comment to lines caused this errors?

Reply:
from typing import cast
        ...
        metrics=[cast(CurveMetric, actual_f_measure.f_measure_per_confidence)]

Reply:
Thanks, changed the way to generate list of expected dashboard_metrics of Performance object returned by get_performance method